src/System.Buffers/src/project.json

```diff
@@ -1,11 +1,16 @@
 {
+  "compilationOptions": {
+    "allowUnsafe": true
+  },
   "dependencies": {
-    "System.Diagnostics.Debug": "4.0.0", 
+    "System.Diagnostics.Debug": "4.0.0",
+    "System.Diagnostics.Tools": "4.0.1-beta-23409",
     "System.Reflection.Primitives": "4.0.0",
     "System.Resources.ResourceManager": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.Extensions": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
+    "System.Slices": "0.1.0-d103015-1",
```

Comment:
Well, you can still reference it from the csproj, I haven't touched that. However, if you add a new contract or break a contract, it will only build from the cli once the nuget package gets updated. This was necessary because we have a "custom" directory structure that the cli doesn't really appreciate just yet. We have an issue tracking adding custom directory structure to dotnet compile.